app/uk/gov/hmrc/tariffclassificationfrontend/models/Queues.scala

```diff
@@ -22,4 +22,12 @@ object Queues {
   val cap = Queue("3", "cap", "CAP")
   val cars = Queue("4", "cars", "Cars")
   val elm = Queue("5", "elm", "ELM")
+
+  val queueNames: Map[String, String] = Map(
```

Comment:
`object Queues` has redundant definitions

Reply:
@pllaurado - we will be able to use this map of queueId to name to display our new event

Reply:
Any reason why this is hard coded? It'd be quite easy to generate

Reply:
Also this is the kind of thing the Queue Service is for

Reply:
Service now provides the queue name